src/helpers/buildFilename.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * @typedef {object} BuildPaperNameParam
+ * @property {string} qualificationName
+ * @property {string} [pageName]
+ * @property {string} [username]
+ * @property {string} [date]
```

Comment:
What is the format of the date?

Reply:
I will rename this property and add a description to the jsDoc.
(This `date` is already formatted)